src/main/java/banking/dao/AccountDAO.java

```diff
@@ -0,0 +1,150 @@
+package banking.dao;
+
+import banking.configuration.DBManager;
+import banking.model.Account;
+import banking.model.Card;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class AccountDAO {
+
+    private final static String INSERT_INTO_ACCOUNT = "INSERT INTO account (number, pin) VALUES (?, ?)";
+    private final static String SELECT_CARD_NUMBER = "SELECT count(number) FROM account WHERE number = (?)";
+    private final static String SELECT_ALL = "SELECT * FROM account";
+    private final static String GET_VALID_CARD = "SELECT * FROM account WHERE number=? AND pin=?";
+    private final static String GET_CARD = "SELECT * FROM account WHERE id = ?";
+    private final static String UPDATE_BALANCE = "UPDATE account SET balance = (balance + ?) WHERE id = ?";
+    private final static String GET_MONEY = "UPDATE account SET balance = (balance + ?) WHERE number = ?";
+    private final static String SEND_MONEY = "UPDATE account SET balance = (balance - ?) WHERE number = ?";
+    private final static String DELETE_ACCOUNT = "DELETE FROM account WHERE id = ?";
+
+    private DBManager dbManager;
+
+    public AccountDAO(DBManager dbManager) {
+        this.dbManager = dbManager;
+    }
+
+    public void insertCard(Account account) {
+        try (PreparedStatement preparedStatement = dbManager.getConnection().prepareStatement(INSERT_INTO_ACCOUNT   )) {
```

Comment:
Please remove the empty space after INSERT_INTO_ACCOUNT.

Reply:
fixed